### PR TITLE
Integrate fork sync

### LINKED ISSUE #8: Integrate fork sync
### Description
To keep repo up-to-date with the upstream changes, the repo must integrates `fork-sync`

### Acceptance Criteria

- Integrate fork sync to the CI pipeline
- Auto approve should be disabled

### Additional Information
https://github.com/marketplace/actions/fork-sync

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?

This PR resolves #8 

### How was it solved?

- [x] Integrate fork sync to the CI pipeline
- [x] Auto approve is disabled by default

### CODE REVIEW

File: .github/workflows/fork-sync.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: tgymnich/fork-sync@v1.8
+        with:
+          owner: base-org
+          head: main
+          base: development
```
Comment (shuse2) [inline .github/workflows/fork-sync.yml]: why are we integrating to the development branch now?

File: .github/workflows/fork-sync.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  schedule:
+    - cron: '0 0 * * *' # once a day
+  workflow_dispatch: # on button click
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: tgymnich/fork-sync@v1.8
```
Comment (sameersubudhi) [inline .github/workflows/fork-sync.yml]: Why don't we use the latest version? CUrrent latest is [v2.0.10](https://github.com/marketplace/actions/fork-sync).
```suggestion
      - uses: tgymnich/fork-sync
```

Comment (sameersubudhi) [reply .github/workflows/fork-sync.yml]: BTW could you please also check https://github.com/wei/pull and any other available options. `wei/pull` seems to be significantly more popular and widely adopted than `tgymnich/fork-sync`.

File: .github/workflows/fork-sync.yml (hunk 1 above)
Comment (nagdahimanshu) [reply .github/workflows/fork-sync.yml]: My understanding from our chat is to integrate `fork-sync` against development branch but I can update it to `main` if it is fine?

**Review by sameersubudhi — CHANGES_REQUESTED** [review]

File: .github/workflows/fork-sync.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        with:
+          owner: base-org
```
Comment (sameersubudhi) [inline .github/workflows/fork-sync.yml:15]:
```suggestion
          owner: base-org
          repo: node
```

File: .github/workflows/fork-sync.yml (hunk 1 above)
Comment (nagdahimanshu) [reply .github/workflows/fork-sync.yml]: Updated it to `main` branch

**Review by sameersubudhi — APPROVED** [review]

**Review by shuse2 — APPROVED** [review]

**Pull request merged**